suite-native/atoms/src/Card/Card.tsx

```diff
@@ -24,23 +25,31 @@ const cardOuterContainerStyle = prepareNativeStyle<{
 }));
 
 const cardInnerContainerStyle = prepareNativeStyle<{
-    isAlertDisplayed: boolean;
+    isAlertTopDisplayed: boolean;
+    isAlertBottomDisplayed: boolean;
```

Comment:
I'd prefer this naming more:
```suggestion
    isTopAlertDisplayed: boolean;
    isBottomAlertDisplayed: boolean;
```

Reply:
opinion: I would make also these two string type, to make sure it always is mutually exclusive (booleans might not be)

Reply:
fa9f32396d52c7d890dd422787c51fcaf7d307d8